src/manage-apps/fix-packages.md

```diff
@@ -0,0 +1,67 @@
+# Cleaning Up Packages with Terminal Commands
+
+If your system complains about a failed upgrade, package manager conflicts, broken upgrades, or other package-related issues, there are several common fixes to these problems. Some package manager issues can be resolved with the graphical update program, but many require the command line.
+
+## General Fixes for Apt Packages
+
+If a package was installed with `apt`, these commands perform general cleanup that can resolve many of `apt`'s errors. These commands should be run one at a time.
+
+```bash
+sudo apt clean
+sudo apt update -m
+sudo dpkg --configure -a
+sudo apt install -f
+sudo apt full-upgrade
+sudo apt autoremove --purge
+```
+
+- `apt clean` - The `clean` command clears out the local repository of retrieved package files.
+- `apt update -m` - The `-m` option ignores corrupted package files during a package update.
+- `dpkg --configure -a` - The `--configure -a` command configures any unpacked but not yet configured packages.
+- `apt install -f` - attempts to correct a system with broken dependencies
```

Comment:
> attempts to correct a system with broken dependencies

This is not a sentence. The rest of these descriptions have a capitalized first word and closing punctuation.

I would say `Attempts to correct broken dependencies.` or `Attempts to resolve unmet dependencies.` because talking about "a system" is more indirect than necessary.